SetGame/Card.swift

```diff
@@ -14,10 +14,10 @@ func randomEnum(_ values: [Any]) -> Any {
 
 enum Shape {
 	case circle
-	case squar
```

Comment:
Probably it's also a good idea to fix the typos in a separate commit to easify the reviews?